.github/workflows/release-builder.yml

```
@@ -49,6 +49,16 @@ env:
   RELEASE_GIT_USER_NAME: "thunderid-automation-bot"
   RELEASE_GIT_USER_EMAIL: "thunderid-bot@wso2.com"
 
+inputs:
```

actionlint (1.7.12)

[error] unexpected key "inputs" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)